src/main/java/org/zalando/planb/revocation/domain/Refresh.java

```diff
@@ -1,41 +1,30 @@
 package org.zalando.planb.revocation.domain;
 
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.immutables.value.Value;
 import org.zalando.planb.revocation.util.UnixTimestamp;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonProperty;
-
-import com.google.auto.value.AutoValue;
-
 /**
- * Represents a notification to refresh all revocations since a point in time.
+ * A notification to refresh all revocations since an instant in time.
  *
- * @author  <a href="mailto:rodrigo.reis@zalando.de">Rodrigo Reis</a>
+ * @author <a href="mailto:rodrigo.reis@zalando.de">Rodrigo Reis</a>
  */
-@AutoValue
+@Value.Immutable
+@JsonSerialize
+@JsonDeserialize(as = ImmutableRefresh.class)
```

Comment:
I'd be great if maybe this class was named "AbstractRefresh" and then the generated class is just named Refresh. The annotation already indicates its immutability,.